Test/AwsCommonRuntimeKitTests/XCBaseTestCase.swift

```diff
@@ -10,7 +10,7 @@ class XCBaseTestCase: XCTestCase {
 
     override func setUp() {
         super.setUp()
-        Logger.initialize(pipe: stdout, level: .trace)
+        Logger.initialize(pipe: stdout, level: .error)
```

Comment:
Did you intend to commit this? I genuinely have no idea what's best, I don't know what it looks like to run tests in this repo

Reply:
I didnt find a good way to enable logs only on failed tests for XCode. Using "trace" mode would spam the outputs here. Therefore, I set it to error. Let me add comments about this, so we could possibly update it in the future.